langchain4j-google-ai-gemini/src/main/java/dev/langchain4j/model/googleai/GoogleAiGeminiStreamingChatModel.java

```diff
@@ -0,0 +1,224 @@
+import java.util.concurrent.ConcurrentHashMap;
+
+import static dev.langchain4j.internal.Utils.copyIfNotNull;
+import static dev.langchain4j.internal.Utils.getOrDefault;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+import static dev.langchain4j.model.googleai.PartsAndContentsMapper.fromMessageToGContent;
+import static dev.langchain4j.model.googleai.SchemaMapper.fromJsonSchemaToGSchema;
+import static java.time.Duration.ofSeconds;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+@Experimental
+@Slf4j
+public class GoogleAiGeminiStreamingChatModel implements StreamingChatLanguageModel {
+    private final GeminiService geminiService;
+    private final String apiKey;
+    private final String modelName;
+    private final Double temperature;
+    private final Integer topK;
+    private final Double topP;
+    private final Integer maxOutputTokens;
+    private final List<String> stopSequences;
+    private final Integer candidateCount;
+    private final ResponseFormat responseFormat;
+    private final GeminiFunctionCallingConfig toolConfig;
+    private final boolean allowCodeExecution;
+    private final boolean includeCodeExecutionOutput;
+    private final Boolean logRequestsAndResponses;
+    private final List<GeminiSafetySetting> safetySettings;
+    private final List<ChatModelListener> listeners;
+
+    @Builder
+    public GoogleAiGeminiStreamingChatModel(String apiKey, String modelName,
+                                            Integer maxRetries,
+                                            Double temperature, Integer topK, Double topP,
+                                            Integer maxOutputTokens, Integer candidateCount,
+                                            Duration timeout,
+                                            ResponseFormat responseFormat,
+                                            List<String> stopSequences, GeminiFunctionCallingConfig toolConfig,
+                                            Boolean allowCodeExecution, Boolean includeCodeExecutionOutput,
+                                            Boolean logRequestsAndResponses,
+                                            List<GeminiSafetySetting> safetySettings,
+                                            List<ChatModelListener> listeners) {
+        this.apiKey = ensureNotBlank(apiKey, "apiKey");
+        this.modelName = ensureNotBlank(modelName, "modelName");
+        this.temperature = getOrDefault(temperature, 1.0);
+        this.topK = getOrDefault(topK, 64);
+        this.topP = getOrDefault(topP, 0.95);
+        this.maxOutputTokens = getOrDefault(maxOutputTokens, 8192);
+        this.candidateCount = getOrDefault(candidateCount, 1);
+        this.stopSequences = getOrDefault(stopSequences, emptyList());
+        this.toolConfig = toolConfig;
+        this.allowCodeExecution = allowCodeExecution != null ? allowCodeExecution : false;
```

Comment:
```suggestion
        this.allowCodeExecution = getOrDefault(allowCodeExecution, false);
```

nit: please use the same style of setting default parameters consistently (and a few places below as well)